Description: Force static build of the inchi library.
 We currently don't want to ship a separate source package nor
 should libinchi be shipped as shared library with openbabel.
 The -fPIC switch is explicitely added to avoid an FTBFS on
 amd64 by relocation error. More information about the latter
 at http://www.cmake.org/pipermail/cmake/2006-March/008482.html.
Author: Michael Banck <mbanck@debian.org>
Last-Update: 2011-02-08

--- a/src/formats/libinchi/CMakeLists.txt
+++ b/src/formats/libinchi/CMakeLists.txt
@@ -10,9 +10,10 @@
 endif(NOT WIN32)
 
 include_directories(${openbabel_SOURCE_DIR}/include/inchi)
-add_library(inchi ${BUILD_TYPE} ${inchi_srcs})
+add_library(inchi STATIC ${inchi_srcs})
 target_link_libraries(inchi ${libs})
 set_target_properties(inchi PROPERTIES
+                            COMPILE_FLAGS -fPIC
                             VERSION ${INCHI_MAJ_VER}.${INCHI_MIN_VER}.${INCHI_PATCH_VER}
                             SOVERSION ${INCHI_MAJ_VER}
                             DEFINE_SYMBOL _USRDLL
